### PR TITLE
Admin filter for auto-confirmation service addresses

### LINKED ISSUE #4683: Add field for Monero Explorer nodes to the Bisq filter

### PR DESCRIPTION
**Pull request by ghost**
A service address can be filtered out using the admin key.  This would be useful if a federated server failed, it could be filtered out so that the auto-confirmation operates with the remaining services.  If all federated service addresses were filtered out the user would have to either confirm manually or use their own explorer service.

example:
filter

Fixes #4683

### CODE REVIEW

File: core/src/main/java/bisq/core/filter/FilterManager.java [hunk 1]
```diff
@@ -398,6 +398,12 @@ public boolean isNodeAddressBanned(NodeAddress nodeAddress) {
                         .anyMatch(e -> e.equals(nodeAddress.getFullAddress()));
     }
 
+    public boolean isAutoConfExplorerBanned(String address) {
+        return getDevFilter() != null &&
```
Comment (chimp1984) [inline core/src/main/java/bisq/core/filter/FilterManager.java]: Here we need to use `getFilter` not `getDevFilter`. DevFilter is the locally stored of the filter admin. Not great naming... should be changed to a more clear name...

**Review by chimp1984 — CHANGES_REQUESTED** [review]

File: core/src/main/java/bisq/core/filter/FilterManager.java (hunk 1 above)
Comment (ghost) [reply core/src/main/java/bisq/core/filter/FilterManager.java]: Argh... thanks for finding that, switched to devFilter for testing: looks like I forgot to revert.

**Review by chimp1984 — APPROVED** [review]
utACK

**Review by ripcurlx — APPROVED** [review]
utACK - Code looks fine.

**Pull request merged**